### ISSUE TITLE
Mac compiles?

### ISSUE THREAD

**Issue by kenpchess** [tryingsomestuff/Minic-Dist#1]
Hello! Minic is quite nice and informative! Is it possible to make a macintosh compile or at least a makefile? Thank you in advance.

-Ken

**Comment by tryingsomestuff** [comment]
Good point. Definitly on my todo list to have a more standard build process.

**Comment by tryingsomestuff** [comment]
But I thought shell scripts are working on MAC OS tool. 
Have you tried the makefile given in the root dir of Minic ? (https://github.com/tryingsomestuff/Minic/blob/master/Makefile)
